<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBoxPageWidget</class>
 <widget class="QWidget" name="ToolBoxPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>43</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#activate_button_{
	background-color:rgba(0, 0, 0, 0);
	border-color: rgba(0, 0, 0, 0);
	border: none;
	height: 24px;
	text-align: left;
	color: white;
	padding-left:4px;
	font: bold;
}

QPushButton#activate_button_:pressed{
	background-color:rgba(0, 0, 0, 0);
	border-color: rgba(0, 0, 0, 0);
	padding-left: 4px;
	border: none;
	color:  rgb(68, 68, 68);
}

QToolButton#close_button_{
	background-color: rgba(0, 0, 0, 0);
	border-radius: 4px;
}

QToolButton#close_button_:pressed{
	background-color: rgb(68, 68, 68);
	border-radius: 4px;
}

QToolButton#help_button_{
	background-color: rgba(0, 0, 0, 0);
	border-radius: 4px;
}

QToolButton#help_button_:pressed{
	background-color: rgb(68, 68, 68);
	border-radius: 4px;
}

QWidget#page_header_{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(0, 0, 0, 0), stop:0.45 rgba(0, 0, 0,30), stop:1 rgba(0, 0, 0, 25));
	/*background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(25, 25, 25, 0), stop:0.753769 rgba(0, 0, 0, 100), stop:1 rgba(0, 0, 0, 84));*/
    border-radius: 3px;
	border: 1px solid black;
}

QWidget#page_background_{
	/*background-color: rgb(255, 128, 0);*/
	background-color: rgb( 237, 149, 31 );
    border-radius: 3px;
}

QWidget#ToolBoxPageWidget{
	background-color: gray;
	border-radius:4px;
}

QWidget#tool_frame_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="page_background_" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="page_header_" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="activate_button_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>21</height>
            </size>
           </property>
           <property name="text">
            <string>Dimensions</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="help_button_">
           <property name="minimumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Resources/AppInterface.qrc">
             <normaloff>:/Images/HelpWhite.png</normaloff>:/Images/HelpWhite.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="close_button_">
           <property name="minimumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Resources/AppInterface.qrc">
             <normaloff>:/Images/CloseWhite.png</normaloff>:/Images/CloseWhite.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="tool_frame_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="tool_frame_layout_">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="url_">
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/AppInterface.qrc"/>
 </resources>
 <connections/>
</ui>
